msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalSdk.java

```diff
@@ -85,6 +89,17 @@ public MsalAuthResult acquireTokenInteractive(@NonNull MsalAuthTestParams authTe
             acquireTokenParametersBuilder.withClaims(authTestParams.getClaims());
         }
 
+        if (authTestParams.getAuthScheme() == AuthScheme.POP) {
```

Comment:
I think this is fine for now. Ultimately we might want to able to supply different values for HTTP Method, claims, URL etc to this API but I understand that would require more changes. So what we've done here is good for now.